test/run_tests.jl

```diff
@@ -67,7 +67,7 @@ solver = solve(
     progress_message = (dt, u, p, t) -> t,
 );
 
-z_centers = parent(Fields.coordinate_field(space))
+z_centers = parent(CC.Fields.coordinate_field(space))
 θ_liq_ice_end = parent(aux.θ_liq_ice)
 q_tot_end = parent(solver.u[end].q_tot)
```

Comment:
Maybe we can just ditch this file, rather than repeating what's in Kid_driver.jl?

Reply:
deleted